playbooks/cloud-provider-openstack-acceptance-test-csi-cinder/run.yaml

```diff
@@ -93,20 +94,19 @@
           fi
           echo "$INSTANCE_UUID" > /var/lib/cloud/data/instance-id
 
-          # Build latest images from source
-          cat << EOF > Dockerfile
-          # Based on centos
-          FROM centos:7.4.1708
-          LABEL maintainers="Kubernetes Authors"
-          LABEL description="Cinder CSI Plugin"
-          # Copy cinderplugin from build directory
-          COPY cinder-csi-plugin /cinderplugin
-          # Install e4fsprogs for format
-          RUN yum -y install e4fsprogs
-          # Define default command
-          ENTRYPOINT ["/cinderplugin"]
-          EOF
-          docker build -t docker.io/k8scloudprovider/cinder-csi-plugin:latest .
+          # Get current branch name, convert to image tag, e.g. release-1.18 to v1.18.0
+          branch="{{ zuul.branch }}"
+          if [[ $branch == "master" ]]; then
+              version=latest
+          else
+              version="v${branch##release-}.0"
+          fi
+          export VERSION=$version
+          make image-csi-plugin
```

Comment:
This command is not supported in master and 1.18 right?

Reply:
@ramineni We just merged that support, didn't we? https://github.com/kubernetes/cloud-provider-openstack/pull/1045 and https://github.com/kubernetes/cloud-provider-openstack/pull/1049

Reply:
Ah, sorry . Totally forgot about that.